partner_credit_limit/models/sale.py

```diff
@@ -12,7 +12,7 @@ class SaleOrder(models.Model):
 
     @api.one
     def check_limit(self):
-        partner = self.partner_id
+        partner = self.partner_invoice_id
```

Comment:
Don't we need a condition like 'checking whether the partner is a company or a contact'?
for 'contact', we dont get accounting fields(credit/debit)...isnt it?